case-lib/config.sh

```diff
@@ -63,7 +58,17 @@ SUDO_LEVEL=${SUDO_LEVEL:-}
 #
 # NO_BT_MODE=true
 
-# Test interval between two test cases, the default value is 5 seconds
-# in CI, which is controlled by sof-framework. In manual run, user can
-# override the default value.
-SOF_TEST_INTERVAL=${SOF_TEST_INTERVAL:-5}
+# SOF_TEST_INTERVAL informs sof-test of how long the external test
+# runner waits between the end of one sof-test and the start of the next
+# sof-test. sof-test uses this value to assign the corresponding kernel
+# logs "no man's land" to the second test, which can be confusing (see
+# for instance https://github.com/thesofproject/sof/issues/5032) but
+# safer: it makes sure no kernel error escapes.
+# See initial review in https://github.com/thesofproject/sof-test/pull/639
+#
+# Note the default value used by Intel's sof-jenkins is 5 seconds, as
```

Comment:
Please note that Jenkins doesn't run any test case, it run case by making use of the test runner, which is sof-framework.

The sof-framework never sets this value, it just wait 5 seconds before it run next test case, that's why we make the initial value 5 seconds, only log within the 1 second before running test case will be collected

Reply:
> The sof-framework never sets this value

Why not? That seems wrong. sof-test should be user friendly, it should not force everyone else and especially not interactive users to configure this because one script in sof-framework is lazy.

>  that's why we make the initial value 5 seconds

You mean the default value?

> only log within the 1 second before running test case will be collected

I fail to parse this part sorry.

Reply:
here you make `SOF_TEST_INTERVAL=${SOF_TEST_INTERVAL:-1}`, the default value is 1, so the log collect point is 4 (- SOF_TEST_INTERVAL - 3)  second before running the test case. but the last test end for CI is 5 second before running the test case, so the log in the 1 second after last test end is not collected in CI.

Reply:
It is a bit ridiculous that the most frequent sof-test user and the one that needs SOF_TEST_INTERVAL the most does not define SOF_TEST_INTERVAL. I filed internal bug 158, kept the default value as is, adjusted the comments and resubmitted all the other changes.